examples/branch-protection/main.tf

```diff
@@ -0,0 +1,60 @@
+
+# Sample organization teams
+resource "github_team" "admin" {
+  name        = "admin-team"
+  description = "Sample admin team"

```

Comment:
Make the names unique so that we don't get the message `unprocessable Message:Name must be unique for this org}]`